src/wasm.rs

```diff
@@ -2,13 +2,17 @@
 use std::ops::Add;
 use std::str;
 
+use flux::ast::*;
```

Comment:
We should avoid these wildcard imports, as they avoid the things that are brought into the namespace. We should either explicitly import what we need or, when there are lots of things we need, import the module itself and reference from the module, e.g. `ast::Node`.